README.md

```diff
@@ -45,6 +45,16 @@ Options:
   -a, --append                Turns on append results to file mode rather than replace
 ```
 
+### Examples:
+  Test the latest underscore module:
+    `citgm underscore@latest`
+
+  Test a local module:
```

Comment:
@MylesBorins it is fully working, perhaps we should add a test to citgm that tests this feature?